spict/R/report.R

```diff
@@ -101,8 +123,12 @@ make.report <- function(rep, reporttitle='', reportfile='report.tex', summaryout
 
     # -- Compile tex file -- #
     #latexcompile <- system(paste('pdflatex -output-directory=../res/', reportfile), intern=TRUE)
-    latexcompile <- system(paste(paste0('pdflatex -output-directory=',file.path(dirname(reportfile))), file.path(reportfile)), intern=TRUE)
```

Comment:
Couldn't we just put the file path and the file name in quotes and remove the double paste?
`latexcompile <- system(paste0('pdflatex -output-directory="', file.path(dirname(reportfile)), '" "', file.path(reportfile), '"'))`